addon/instance-initializers/load-feature-controls.js

```diff
@@ -0,0 +1,18 @@
+import config from 'ember-get-config';
+const { featureControls } = config;
+
+export function initialize(appInstance) {
+  const features = appInstance.lookup('service:features');
+  if (featureControls.useLocalStorage) {
+    const flags = JSON.parse(window.localStorage.getItem('storage:feature-controls'));
```

Comment:
instead of using `JSON.parse` and `window.localStorage`, you should be able to get an instance of ember-local-storage with `getStorage`, see https://github.com/funkensturm/ember-local-storage/blob/master/addon/helpers/storage.js#L146

Reply:
sadly, `getStorage` doesn't do much as it basically call `tryStorage` which basically does this: https://github.com/funkensturm/ember-local-storage/blob/master/addon/helpers/storage.js#L21

It only alllows to not access the localStorage directly but we still need to parse the string :s

`const flags = JSON.parse(getStorage('local')['storage:feature-controls']);`

Reply:
here is the resulting code (pasting here because I'm not sure we'll keep the commit):
```js
export function initialize(appInstance) {
  const features = appInstance.lookup('service:features');
  if (featureControls.useLocalStorage) {
    let featureControlsJSON = getStorage('local')['storage:feature-controls'];
    if (featureControlsJSON) {
      const flags = JSON.parse(featureControlsJSON);
      if (flags) {
        Object.keys(flags).forEach(flag => {
          flags[flag] ? features.enable(flag) : features.disable(flag);
        });
      }
    }
  }
}
```

Reply:
I'm not convinced wither...
If you're okay with the current code in the PR, let's just go with it, I wouldn't want to block the feature for something not critical